db/migrate/20230116181439_create_work_processes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkProcesses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :work_processes, id: :uuid do |t|
+      t.string :title
+      t.string :description
+      t.references :occupation_standard, null: false, foreign_key: true, type: :uuid
+      t.integer :default_hours
+      t.integer :minimum_hours
+      t.integer :maximum_hours
```

Comment:
@jenlu33 could you make this a range like you did for the `hybrid_hours` field on the occupations table? That way we can more easily query work processes that fall within a given search range.